docs/hosting/securing/set-up-ssl.md

```diff
@@ -0,0 +1,25 @@
+---
+#https://www.notion.so/n8n/Frontmatter-432c2b8dff1f43d4b1c8d20075510fe4
+description: "Set up SSL for your self-hosted n8n instance."
+contentType: howto
+---
+
+# Set up SSL
```

Comment:
The preferred way of setting up TLS is by using a reverse proxy like Traefik which also takes care of certificate renewals.

Reply:
I agree. We could maybe create 2 sections here:
1. using a reverse proxy to outsource TLS management on ingress, or
2. passing in certs to n8n directly

We should then mark 1 as "Recommended", and 2 as "Not Recommended".

I'd also be fine with us skipping 2, and deprecating that option in one of the future releases.

Reply:
I would add both options to the docs just to be complete, I am also not sure on removing the option from the application if anything we should be creating certs by default given that we don't allow http outside of localhost anymore.

We should still make sure that using a reverse proxy or NLB is marked clearly as preferred.

Reply:
Thanks for the feedback, everyone.